docs/irq-load-balancing.md

```diff
@@ -58,4 +59,137 @@ spec:
+     cpu-quota.crio.io: "true"
+spec:
+  containers:
+  - name: dynamic-irq-pod
+    image: "quay.io/openshift-kni/cnf-tests:4.5"
+    command: ["sleep", "10h"]
+    resources:
+      requests:
+        cpu: 2
+        memory: "200M"
+      limits:
+        cpu: 2
+        memory: "200M"
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  runtimeClassName: dynamic-irq-profile
+
+```
+1. Ensure both annotations exist.
+2. Ensure the pod has its ```runtimeClassName``` as the respective profile name, in this case dynamic-irq-profile.
+3. Ensure the node selector targets a cnf-worker.
+
+Ensure the pod is running correctly.
+```
+oc get pod -o wide
+NAME              READY   STATUS    RESTARTS   AGE     IP             NODE                  NOMINATED NODE   READINESS GATES
+dynamic-irq-pod   1/1     Running   0          5h33m   10.135.1.140   cnf-worker.demo.lab   <none>           <none>
+```
+1. Ensure status is ```running```.
+2. Ensure the pod is scheduled on a cnf-worker node, in our case on the ```cnf-worker.demo.lab``` node.
+
+Find out the CPUs dynamic-irq-pod runs on.
+```
+oc exec -it dynamic-irq-pod -- /bin/bash -c "grep Cpus_allowed_list /proc/self/status | awk '{print $2}'"
+Cpus_allowed_list:	2-3
+```
+
+Ensure the node configuration is applied correctly.
+Connect to the ```cnf-worker.demo.lab``` node to verify the configuration.
+```
+oc debug node/ocp47-worker-0.demo.lab
+Starting pod/ocp47-worker-0demolab-debug ...
+To use host binaries, run `chroot /host`
+
+Pod IP: 192.168.122.99
+If you don't see a command prompt, try pressing enter.
+
+sh-4.4# 
+```
+Use the node file system:
+```
+sh-4.4# chroot /host
+sh-4.4# 
+```
+
+1. Ensure the default system CPU affinity mask does not include the dynamic-irq-pod CPUs, in our case 2,3.
+```   
+cat /proc/irq/default_smp_affinity
+33
+```
+
+2. Ensure the system IRQs are not configured to run on the dynamic-irq-pod CPUs
+```
+find /proc/irq/ -name smp_affinity_list -exec sh -c 'i="$1"; mask=$(cat $i); file=$(echo $i); echo $file: $mask' _ {} \;
+/proc/irq/0/smp_affinity_list: 0-5
+/proc/irq/1/smp_affinity_list: 5
+/proc/irq/2/smp_affinity_list: 0-5
+/proc/irq/3/smp_affinity_list: 0-5
+/proc/irq/4/smp_affinity_list: 0
+/proc/irq/5/smp_affinity_list: 0-5
+/proc/irq/6/smp_affinity_list: 0-5
+/proc/irq/7/smp_affinity_list: 0-5
+/proc/irq/8/smp_affinity_list: 4
+/proc/irq/9/smp_affinity_list: 4
+/proc/irq/10/smp_affinity_list: 0-5
+/proc/irq/11/smp_affinity_list: 0
+/proc/irq/12/smp_affinity_list: 1
+/proc/irq/13/smp_affinity_list: 0-5
+/proc/irq/14/smp_affinity_list: 1
+/proc/irq/15/smp_affinity_list: 0
+/proc/irq/24/smp_affinity_list: 1
+/proc/irq/25/smp_affinity_list: 1
+/proc/irq/26/smp_affinity_list: 1
+/proc/irq/27/smp_affinity_list: 5
+/proc/irq/28/smp_affinity_list: 1
+/proc/irq/29/smp_affinity_list: 0
+/proc/irq/30/smp_affinity_list: 0-5
+```
+Note: Some IRQ controllers do not support IRQ re-balancing and will always expose the online CPUs as the IRQ mask.
```

Comment:
`will always expose the online CPUs` -> will always expose all online CPUs

Reply:
done